src/edu/ucsb/nceas/metacat/dataone/CNodeService.java

```diff
@@ -124,7 +125,7 @@ public static CNodeService getInstance(HttpServletRequest request) {
    */
   private CNodeService(HttpServletRequest request) {
     super(request);
-    logMetacat = Logger.getLogger(CNodeService.class);
+    logMetacat = LogFactory.getLog(CNodeService.class);
```

Comment:
Hrrm. It shows the code is `logMetacat = LogFactory.getLog(CNodeService.class);` rather than `logMetacat = Logger.getLogger(CNodeService.class);`. I don't why we see the different code. Anyway, I initialize the log object in the class field rather than the constructor. See the commit 490379dfb12641f226de50ba9e7e825774b4ca50